Makefile.core.mk

```diff
@@ -344,6 +344,11 @@ shellcheck:
 MARKDOWN_LINT_WHITELIST=localhost:8080,storage.googleapis.com/istio-artifacts/pilot/,http://ratings.default.svc.cluster.local:9080/ratings
 
 lint_modern: lint-python lint-copyright-banner lint-scripts lint-dockerfiles lint-markdown lint-yaml
+	@bin/check_helm.sh
+	@GOARCH=amd64 GOOS=linux bin/check_samples.sh
+	@GOARCH=amd64 GOOS=linux bin/check_dashboards.sh
+	@GOARCH=amd64 GOOS=linux go run mixer/tools/adapterlinter/main.go ./mixer/adapter/...
+	@GOARCH=amd64 GOOS=linux bin/check_pilot_codegen.sh
 	@golangci-lint run -c ./common/config/.golangci.yml ./galley/...
```

Comment:
nit: for the future, the most useful order would be to do the things that are fast and fail most often first. So checking licenses (very fast, easy to break when you add files), and linting go (most often failure) first, then things like checking links that is somewhat slow and probably doesn't produce errors as often last